APR-Core/Config.lua

```diff
@@ -181,6 +181,15 @@ function APR.settings:createBlizzOptions()
                     )
                 end
             },
+            statusButton = {
+                order = 1.5,
+                name = "Status",
```

Comment:
don't we put this in CF for translation and just not translate the frame content ?.
but do it for the buttons?